docs/guides/feature-flags.rst

```diff
@@ -27,6 +27,18 @@ In case you prefer to use the latest ``conda`` version available, this is the fl
 Makes Read the Docs to install all the requirements at once on ``conda create`` step.
 This helps users to pin dependencies on conda and to improve build time.
 
+``CONDA_USES_MAMBA``: :featureflags:`CONDA_USES_MAMBA`
+
+``conda`` solver consumes 1Gb minimum when installing any package using ``conda-forge`` channel.
```

Comment:
I think you mean resolver

```suggestion
Conda's resolver consumes 1Gb minimum when installing any package using ``conda-forge`` channel.
```

Reply:
Well... I think it's the same, but they call it "Solver" :) 

https://docs.conda.io/projects/conda/en/latest/api/solver.html#conda.core.solve.Solver